test/powershell/Modules/Microsoft.PowerShell.Core/CompatiblePSEditions.Module.Tests.ps1

```diff
@@ -473,8 +473,8 @@ Describe "Additional tests for Import-Module with WinCompat" -Tag "Feature" {
 
         It "Verify that Warning is generated with default WarningAction" {
             $LogPath = Join-Path $TestDrive (New-Guid).ToString()
-            & $pwsh -NoProfile -NonInteractive -c "[System.Management.Automation.Internal.InternalTestHooks]::SetTestHook('TestWindowsPowerShellPSHomeLocation', `'$basePath`');Import-Module $ModuleName" *> $LogPath
-            $LogPath | Should -FileContentMatch 'loaded in Windows PowerShell'
+            & $pwsh -NoProfile -NonInteractive -c "[System.Management.Automation.Internal.InternalTestHooks]::SetTestHook('TestWindowsPowerShellPSHomeLocation', `'$basePath`');Import-Module $ModuleName;Get-Error" *> $LogPath
```

Comment:
Why `Get-Error` is needed now?

Reply:
Needed this to figure out why this was failing.  It only shows up when the test fails.  Needed the stacktrace.

Reply:
So, I guess if the test run successfully, `Get-Error` shouldn't return anything, which means only the expected warning message is written to the stdout stream?

Reply:
yes